compose_generator/odcs_fetcher.py

```diff
@@ -1,7 +1,12 @@
 """Fetch ready ODCS compose"""
+import tempfile
 from dataclasses import dataclass
 from pathlib import Path
+from typing import Optional
 
+import requests  # type: ignore
```

Comment:
I think there's a way to avoid this ingore:
```
Library stubs not installed for "requests"Mypy[import-untyped](https://mypy.readthedocs.io/en/latest/_refs.html#code-import-untyped)
Hint: "python3 -m pip install types-requests"
(or run "mypy --install-types" to install all missing stub packages)
See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-importsMypy[note](https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports)
```

Reply:
Worked locally, added the line to the workflow, let's see if it works as expected (I do wonder by how much will it slow the tests)

Reply:
As far as I see it, it might slow down the type checking, but just because it's skipping it otherwise.